junit-commons/src/main/java/org/junit/gen5/commons/util/ReflectionUtils.java

```diff
@@ -405,23 +405,7 @@ private static void handleException(Throwable ex) {
 		if (ex instanceof Error) {
 			throw (Error) ex;
 		}
-		// TODO Research if throwing the exception itself would be a better option
-		throw new TargetExceptionWrapper(ex);
-	}
-
-	public static class TargetExceptionWrapper extends RuntimeException {
-
-		private static final long serialVersionUID = 3733792088719661853L;
-
-		private final Throwable targetException;
-
-		private TargetExceptionWrapper(Throwable targetException) {
-			this.targetException = targetException;
-		}
-
-		public Throwable getTargetException() {
-			return targetException;
-		}
+		throw ex;
```

Comment:
Now we don't need the two `ifs` checking for `RuntimeException` and `Error` anymore, do we?